docusaurus.config.js

```diff
@@ -80,6 +80,18 @@ const config = {
         'Licensed by ComCODE with <a href="https://creativecommons.org/licenses/by-nc-sa/4.0/" target="_blank" rel="noopener noreferrer">CC BY-NC-SA 4.0</a> for creative and educational use.',
     },
   },
+
+  // Global markdown config
```

Comment:
Is this required for upgrade?

Reply:
Not strictly required. The Docusaurus migration guide does however [recommend](https://docusaurus.io/docs/migration/v3#turn-off-mdx-v1-compat) configuring the MDX1 compatibility options, so I have created the markdown config for that reason. Other fields are included in the config but do not change current behaviour.